### ISSUE TITLE
count(distinct null) = null

### ISSUE THREAD

**Issue by 747646769** [ClickHouse/ClickHouse#3214]
#### SQL
``` SQL
SELECT count(NULL);
SELECT count(DISTINCT NULL);
```
#### Expect
```
0
0
```

#### Actual
```
0
NULL
```

**Comment by deangelo-llooker** [comment]
Observed with version 19.15.2.2 as well.

**Comment by alexey-milovidov** [comment]
Fixed in #11661.

**Comment by alexey-milovidov** [comment]
Released in 20.5.
